src/tfvc/tfvc-extension.ts

```diff
@@ -51,33 +51,46 @@ export class TfvcExtension  {
             "Checkin");
     }
 
-    public async Exclude(resource?: Resource): Promise<void> {
+    public async Exclude(resources?: Resource[]): Promise<void> {
         this.displayErrors(
             async () => {
-                if (resource) {
+                if (resources) {
```

Comment:
slight optimization - check for resources.length > 0 to avoid the await below in that case